aws-test/tests/aws_securityhub_standards_subscription/test-list-expected.json

```diff
@@ -3,7 +3,7 @@
     "enabled_by_default": true,
     "name": "CIS AWS Foundations Benchmark v1.2.0",
     "standards_arn": "arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0",
-    "standards_status": "READY",
+    "standards_status": "INCOMPLETE",
```

Comment:
Since these output parameters will keep on varying depending on the terraform and the provider version, I feel it is better to remove these parameters from the expected.json file.